### PR TITLE
Toast  component

### PR DESCRIPTION
**Pull request by neetukumari4858**
I added Toast component and I did some other changes that is related to previous component.

### CODE REVIEW

File: components/Avtar.css [hunk 1]
```diff
@@ -1,116 +1,7 @@
...
-.Anchor-link{
-    text-decoration: none;
-    font-size: 1.2rem;
-    color: black;
-
-}
-.list{
-    margin-top: -1rem;
-}
+@import url("/document.css");
```
Comment (abhijitdotsharma) [inline components/Avtar.css:1]: There is a typo on your filename it should be Avatar.css

**Review by abhijitdotsharma — CHANGES_REQUESTED** [review]
One question - we normally use element-property instead of elementProperty in CSS. Not a hard rule I guess but will make the code review easy.
Rest Looks good to me

File: index.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+#text{
+    height: 17rem;
+    width: 40rem;
+    font-size: 1.3rem;
+    text-align: center;
+    margin-left: 20rem;
+    margin-top: 4rem;
+}
+#getStartedbtn{
```
Comment (Manju-sree) [inline index.css]: better to use class than id for styles

**Review by Manju-sree — CHANGES_REQUESTED** [review]
rest looks good to me, please change id's to class

**Review by Manju-sree — APPROVED** [review]
looks good to merge

**Pull request merged**